source/mainnet/smart-contracts/tutorials/wCCD/wCCD-front-end-set-up.rst

```diff
@@ -4,6 +4,354 @@
+    ``Chrome`` browser.
+
+    .. image:: ./images/wCCD_tutorial_12.png
+        :width: 100 %
+
+    .. note::
+
+        The puzzle icon at the top right of the ``Chrome`` browser allows you to manage your browser extensions.
+        You can enable pinning of the Concordium browser wallet.
+
+        .. image:: ./images/wCCD_tutorial_13.png
+            :width: 30 %
+
+    .. dropdown:: Getting your private key from an account already imported to the `concordium-client`
+
+        Display your keys with the following :ref:`command <concordium-client-display>`
+
+        .. code-block:: console
+
+            $./concordium-client config show
+
+        Save the ``encryptedSignKey`` blob to a file named ``output.json``. The content of that file
+        should look similar to the below content.
+
+        .. code-block:: json
+
+            {
+                "cipherText": "K1ylur5Qy+UUYlwyShu1W6rRgRhcN12e91SEGZ9UBboEzTVVQ80cDpsJNBQmU+sBlo1FKrGxKFzPjxhKxxohmZ99yDXgyo9bMDxuTosqcfY=",
+                "metadata": {
+                    "encryptionMethod": "AES-256",
+                    "initializationVector": "oJhcClLqUEotJxh4nmuCgA==",
+                    "iterations": 100000,
+                    "keyDerivationMethod": "PBKDF2WithHmacSHA256",
+                    "salt": "0XSYLtrsLN+XXwYqxD+gDw=="
+                }
+            }
+
+        Download the :ref:`utils tool <downloads-testnet-auxiliary-tools>` under the auxiliary tools section.
+        This tool is able to decode your encrypted key.
+
+        You can find additional information on the `utils` tool :ref:`here <developer-tools>`.
+
+        Decode your private key by running the decrypt command on the ``output.json`` file.
+        You will need to enter your password from the backup file when it was exported from the mobile wallet.
+
+        .. code-block:: console
+
+            $./utils decrypt --in output.json --out decrypted.example
+
+        Your private key will be saved to the ``decrypted.example`` file.
+
+    Enter the below ``JSON-RPC`` endpoint into the browser wallet to connect to
+    your local ``JSON-RPC server`` on port 9095.
+
+    .. code-block:: console
+
+        http://127.0.0.1:9095
+
+    In case you run an older pre-MVP wallet, you have to enter the private key and
+    the associated account into the browser wallet similar to the below string. In case you run a newer pre-MVP wallet,
+    you can create a new account with the associated private key in the browser wallet.
+
+    .. code-block:: toml
+
+        74ff83a13ca066298583dcb9151822359fd2e4c9b69c9ca427455da565f6129b,3oLNhuxM7yrf3LrJa3hH5NfocTViGS8Aj2t6YScWNvUq4o2nC
+
+    You completed the browser wallet setup. Check that your account balance is displayed and you have enough
+    CCD to be able to execute transactions.
+
+    .. note::
+        You are connected to a website with your browser wallet when you see the green ``Connected`` button.
```

Comment:
I think we should put a red box around the connected button in the image, so it is easier to find.